lib/codecs/src/decoding/framing/chunked_gelf.rs

```diff
@@ -383,27 +516,65 @@ impl Decoder for ChunkedGelfDecoder {
     }
 }
 
+// TODO: should I add compression tests in the UDP socket module tests?
+// as we did with the initial chunked gelf implementation.
 #[cfg(test)]
 mod tests {
```

Comment:
Should I add compressed GELF tests in the socket source in addition to those unit tests added here? this would be more of an "integration test", where we will also test that the Gelf's format deserializer also works with compressed chunked gelf payloads.

This would be very similar to what has been done in https://github.com/vectordotdev/vector/blob/983a9937ac5fc24db0203eec0390550351ec2cbe/src/sources/socket/mod.rs#L1137

But I don't know if it would be worth, as doing only unit testing here seems reasonable